usaspending_api/awards/v2/filters/view_selector.py

```diff
@@ -344,10 +344,12 @@ def spending_by_category(category, filters):
         view_chain = ['SummaryNaicsCodesView']
     elif category == 'cfda':
         view_chain = ['SummaryCfdaNumbersView']
-    elif category in ['county', 'district']:
+    elif category in ['county', 'district', 'state/territory', 'country']:
```

Comment:
Does `state/territory` work? or should it be `state_territory`?

Reply:
Def should be `state_territory`, fixing that.